ui/app/pages/create-account/connect-hardware/index.js

```diff
@@ -243,11 +245,12 @@ class ConnectHardwareForm extends Component {
         selectedPath={this.props.defaultHdPaths[this.state.device]}
         device={this.state.device}
         accounts={this.state.accounts}
-        selectedAccount={this.state.selectedAccount}
+        connectedAccounts={this.props.connectedAccounts}
+        selectedAccounts={this.state.selectedAccounts}
```

Comment:
It looks like this prop is ignored :thinking: I'm guessing this state was migrated from one component to the other, but the migration wasn't finished?

Reply:
I don't quite understand this comment but I just rewrote all of the selection logic to make it way more simple.